test/api/v3/unit/libs/cron.test.js

```diff
@@ -143,15 +144,16 @@ describe('cron', () => {
       expect(user.purchased.plan.consecutive.trinkets).to.equal(1);
     });
 
-    it('increments plan.consecutive.gemCapExtra when user has reached a month that is a multiple of 3', () => {
+    it('increments plan.consecutive.gemCapExtra when one-month-recurring subscription has reached a month that is a multiple of 3', () => {
+      // TODO if approved, delete this because the new tests below include this situation
       user.purchased.plan.consecutive.count = 5;
-      user.purchased.plan.consecutive.offset = 1;
       cron({user, tasksByType, daysMissed, analytics});
       expect(user.purchased.plan.consecutive.gemCapExtra).to.equal(5);
-      expect(user.purchased.plan.consecutive.offset).to.equal(0);
+      expect(user.purchased.plan.consecutive.count).to.equal(6);
     });
 
     it('increments plan.consecutive.gemCapExtra multiple times if user has been absent with continuous subscription', () => {
+      // TODO if approved, delete this because the new tests below include this situation
```

Comment:
what about these comments @Alys ?

Reply:
It's up to you. If you're happy with that the new tests cover what these old tests cover, and if you don't want to keep the old tests because they're related to other tests around them, then I can remove them now.

Reply:
ah ok, let's delete them then!

Reply:
I've deleted those four tests.